src/iceqube/storage.py

```diff
@@ -78,22 +79,10 @@ def set_sqlite_pragmas(self):
 
         :return: None
         """
-
-        def _pragmas_on_connect(dbapi_con, con_record):
-            """
-            Set pragmas on every connection, as it is the only way to guarantee
-            that they are set.
-            However, some pragmas, like the wal journal_mode we use below
-            will generate errors if a transaction is currently open on the
-            database - so try to do this, but catch and pass if it produces
-            an error.
-            """
-            try:
-                dbapi_con.execute("PRAGMA journal_mode = WAL;")
-            except OperationalError:
-                pass
-
-        event.listen(self.engine, "connect", _pragmas_on_connect)
+        try:
+            self.engine.execute("PRAGMA journal_mode = WAL;")
+        except OperationalError:
+            pass
```

Comment:
Should this perhaps raise some sort of very big warning? Would very much like to know in case this fails :)

Reply:
I know it didn't before... will open an issue...